exchanges/orderbook/orderbook_types.go

```diff
@@ -86,7 +86,20 @@ type Base struct {
 	Pair     currency.Pair
 	Asset    asset.Item
 
-	LastUpdated  time.Time
+	// When a change occurs on the exchange books. This does not mean the
+	// change is out of sync with the exchange.
```

Comment:
It's just to let people know that the time is not representative of the difference between when this change occurred on the books to you receiving this change. For example this time could have occurred an hour ago but because there has been no change on the books, it's still appropriate to trade on it.

So the extra fields are a more of a distinction between the times given by the exchange. If you want me to expand on the commentary I am happy to.